src/main/java/seedu/address/logic/Messages.java

```diff
@@ -9,7 +9,7 @@
 import seedu.address.model.person.Person;
 
 /**
- * Container for user visible messages.
+ * Container for user and event visible messages.
```

Comment:
I think over here "user visible messages" refer to messages that are visible to the user?

Reply:
Oh my bad! I misunderstood the sentence. I will change it!